kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# Note the v1 in the filename refers to the version of the CustomResourceDefinition
+# not any of the versions of API being defined.
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: kafkaproxyingresses.kroxylicious.io
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: kroxylicious.io
+  scope: Namespaced
+  names:
+    plural: kafkaproxyingresses
+    singular: kafkaproxyingress
+    kind: KafkaProxyIngress
+    shortNames:
+      - kpi
+  # list of versions supported by this CustomResourceDefinition
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: { }
+      schema:
+        openAPIV3Schema:
```

Comment:
Please please please can we add `description` to all the properties in the schema.